mars/deploy/oscar/ray.py

```diff
@@ -39,12 +44,134 @@
 DEFAULT_SUPERVISOR_SUB_POOL_NUM = 0
 
 
-def _load_config(filename=None):
+def _load_config(config: Union[str, Dict] = None):
     # use default config
-    if not filename:  # pragma: no cover
+    if isinstance(config, str):
+        filename = config
+    else:
         d = os.path.dirname(os.path.abspath(__file__))
         filename = os.path.join(d, 'rayconfig.yml')
-    return load_service_config_file(filename)
+    full_config = load_service_config_file(filename)
+    if config and not isinstance(config, str):
+        if not isinstance(config, Dict):
+            raise ValueError(f'{config} is not a dict')
+        flatten_keys = set(k for k in config.keys() if isinstance(k, str) and '.' in k)
+        nested_flatten_config = flatten_dict_to_nested_dict({k: config[k] for k in flatten_keys})
```

Comment:
Why stop using original config format?

Reply:
Json/dict config is still usable. Sometimes json or dict config is not very intuitive, so I add properties config support.

Reply:
A config like 
```
config={
    'scheduling.autoscale.enabled': True,
    'scheduling.autoscale.scheduler_check_interval': 1,
    'scheduling.autoscale.scheduler_backlog_timeout': 1,
    'scheduling.autoscale.worker_idle_timeout': 10,
    'scheduling.autoscale.min_workers': 1,
    'scheduling.autoscale.max_workers': 4
}
```
is more readable than flooing config for me:
```
config={
    'scheduling': {
        "autoscale": {
            "enable": True,
            "scheduler_check_interval": 1,
            "scheduler_backlog_timeout": 1,
            "worker_idle_timeout": 1,
            "min_workers": 1,
            "max_workers": 4
        }
    }
}
```

Anyway, it's just a preference. If this is not recommanded, I can remove it.